.release/security-scan.hcl

```diff
@@ -29,7 +29,8 @@ binary {
   triage {
     suppress {
       vulnerabilities = [
-        "GO-2022-0635", // github.com/aws/aws-sdk-go@v1.55.5 TODO(dduzgun-security): remove when deps is resolved
+        "GO-2025-3408",        // github.com/hashicorp/yamux@v0.1.2 TODO(jrasell): remove when dep updated.
+        "GHSA-29qp-crvh-w22m", // github.com/hashicorp/yamux@v0.1.2 TODO(jrasell): remove when dep updated.
```

Comment:
LGTM! Only the GO identifier should be enough I think but we can keep both for not doing back and forward work considering the CI takes a long time to run. Also, thanks for the bump and cleanup of aws-sdk 🙏